corehq/ex-submodules/phonelog/models.py

```diff
@@ -19,6 +19,9 @@ class DeviceReportEntry(models.Model):
 
     class Meta:
         unique_together = [('xform_id', 'i')]
+        index_together = [
+            ("date", "domain"),
```

Comment:
Hey that logic makes sense to me. My gut was to use domain as the major, but when I saw the query planner I changed my mind. When you reason about it your way I think that's the best route